sql-cli/sql_cli/__main__.py

```diff
@@ -91,18 +91,12 @@ def validate(
     """
 )
 def run(
-    directory: Path = typer.Argument(
+    workflow_directory: Path = typer.Argument(
         default=...,
         show_default=False,
         exists=True,
         help="directory containing the sql files.",
     ),
-    dags_directory: Path = typer.Argument(
-        default=...,
-        show_default=False,
-        exists=True,
-        help="directory for the generated DAGs.",
-    ),
     connection_file: Path = typer.Option(
```

Comment:
We should be consistent with the other commands, which use `environment`, and we retrieve the connection based on the environment path containing configurations.